### ISSUE TITLE
menu error

### ISSUE THREAD

**Issue by Ari1009** [PotatoDoge/PygamePokemon#1]
the game only open till menu after that it is stuck is there any fix

**Comment by PotatoDoge** [comment]
it is not a bug at all, I am still working on the game itself. I already have it finished , but I am cleaning the code up a little bit. I'll upload the rest of the game in a week or so, I'm sure you'll like it ;)

**Comment by Ari1009** [comment]
Thanks a lot for your reply.so after a week i can play the game
right?because as of now i am able to open menu screen and not able to go
further so with the next update i can play the game right?
i also wanted to ask you a favour can you make a tutorial about how you
program what each ,modeule do in the game  i want to learn coding more so
it will be really helpful if u can do something about it.
THANKS A LOT

**Comment by PotatoDoge** [comment]
Yeah, I'll probably have the game finished by the end of this month , tops. Obviously it is a simple and short game compared to the original ones. It is made in python with the pygame library. And yeah, I actually have a youtube channel where I teach Java from 0, and when I finish that course, I'll probably start with one about python(and probably one about pygame). Here's the link: https://www.youtube.com/watch?v=otWz8VDPRww&t=27s   
Thanks for showing interest in this small project, i really appreciate it. Best regards!
